Align documentation of connection management with connectivity concepts publication (#2992)
nest/nest-simulator: doc/htmldoc/synapses/handling_connections.rst

.. _handling_connections:

> How to handle created connections

Reviewer: ```suggestion
How to inspect and modify connections
```

> How to handle created connections
> =================================

Reviewer: ```suggestion
How to inspect and modify connections
==============================
```
Also in the text below, "already created" sounds strange. Either write "existing connections" or just "connections", because when you talk about inspecting and modifying, that implies that the connections must exist.

Here, we describe how to inspect and modify already created connections.

> For creating new connections, refer to :ref:`connection_management`, and for parameterizing synapses, see :ref:`synapse_spec`.

Reviewer: The parameterization you mean here is the one during creation, isn't it? Then the sentence is confusing. Maybe something like
```suggestion
:ref:`connection_management` describes how to create new connections and :ref:`synapse_spec` how to parameterize synapses as they are created.
```
